shared/clinical-code-sets/medications/serotonin-modulator-stimulator/1/README.md

```diff
@@ -6,10 +6,10 @@ This code set was created from getset.ga, using the following list from the PI o
 
 ## Prevalence log
 
-By examining the prevalence of codes (number of patients with the code in their record) broken down by clinical system, we can attempt to validate the clinical code sets and the reporting of the conditions. Here is a log for this code set. The prevalence range `... - ...%` suggests that this code set is well defined.
+By examining the prevalence of codes (number of patients with the code in their record) broken down by clinical system, we can attempt to validate the clinical code sets and the reporting of the conditions. Here is a log for this code set. The prevalence range `0.001 - 0.003%` suggests that this code set is well defined.
```

Comment:
The narrow range might suggest it is well defined, but with only 40 patients in the whole of GM with this in their record I'd say it's either missing codes, or is something that isn't prescribed in primary care. You get some meds that only ever get prescribed in hospital.

Reply:
I see. The finding-missing-code script was used to look for potential missing codes. What should we do with this medication group then? Do we need to note with the PI?

Reply:
Yes - let the PI know. Show them the codes you have, and ask if they would expect more than 40 people in GM to have ever had this prescribed by a GP.

Reply:
OK I just email the PI.